lib/db/reasoning.js

```diff
@@ -0,0 +1,117 @@
+const { fetch } = require('../fetch');
+
+const { httpBody } = require('../response-transforms');
+
+const jsonify = res => {
+  res.headers.append('content-type', 'application/json');
+  return res;
+};
+
+const consistency = (conn, database, options, params) => {
```

Comment:
I think you need `options = {}` here. If we are saying this argument is optional, and it is, then you shouldn't have to pass `{}` to keep the function from crashing.